content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -230,8 +211,12 @@ against a list of strings in the JWT token.
     EOF
     {{< /text >}}
 
-1.  Verify that the connection to the `httpbin`
-service succeeds after the RBAC policy has become effective.
+Wait for the newly defined RBAC policy to take effect.
```

Comment:
Please indent the text to maintain the numbering on the list. Four spaces are needed in front of lines 214-216.

Reply:
Done.